module/dice/dice.mjs

```diff
@@ -110,8 +110,9 @@ export async function d20Roll({
  * @param {boolean} [config.disadvantage]  Is something granting this roll disadvantage?
  * @param {boolean} [config.fastForward]   Should the roll dialog be skipped?
  * @returns {{isFF: boolean, advantageMode: number}}  Whether the roll is fast-forward, and its advantage mode.
+ * @internal
  */
-function _determineAdvantageMode({event, advantage=false, disadvantage=false, fastForward}={}) {
+export function _determineAdvantageMode({event, advantage=false, disadvantage=false, fastForward}={}) {
```

Comment:
Instead of exporting a function, can we please reframe this as a static method on the `D20Roll` class?